sv.h

```diff
@@ -746,6 +746,40 @@ Unsets the PV status of an SV.
 Tells an SV that it is a string and disables all other C<OK> bits.
 Will also turn off the UTF-8 status.
 
+=for apidoc Am|U32|SvPIOK|SV* sv
```

Comment:
I like the idea of this family of test/set macros, but unfortunately the naming is problematic.

We already have a set of macros `SvNIOK` that test if an SV has `NOK` **or** `IOK`, so to have a similarly-named one `SvPIOK` which tests for both `POK` **and** `IOK` is going to cause confusion. Is there some way we can work in the words "and" or "both" or somesuch into its name?

Reply:
A better name, even if the font github uses makes it look like the word "land". In a real code editor with a proper font I'm sure it looks fine.